snapshot-labs/snapshot
# refactor(env): move brovider url to env file

### Changes 
1. Moves brovider url to env file so it makes it more flexible to configure

### How to test
1. Try to login
2. Try to create a space/proposal
3. Try to vote
Everything should work as before

### To-Do


### Self-review checklist

### Additional notes or considerations
*_(Include any other relevant information or context that may be helpful for the reviewer)_*
